libraries/AP_Mount/AP_Mount_MAVLink.h

```diff
@@ -1,19 +1,19 @@
 /*
-  Gremsy mount backend class
+  MAVLink Gimbal Protocol v2 backend class
  */
 #pragma once
 
 #include "AP_Mount_config.h"
 
-#if HAL_MOUNT_GREMSY_ENABLED
+#if HAL_MOUNT_GREMSY_ENABLED || HAL_MOUNT_AVT_ENABLED
```

Comment:
```suggestion
#if HAL_MOUNT_MAVLINK_ENABLED
```

Reply:
we've renamed the backend to MAVLink so we no longer have the AVT specific definition